src/main/java/duke/exception/DukeInvalidDateFormatException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+/**
+ * Handles invalid date.
+ */
+public class DukeInvalidDateFormatException extends DukeException {
+    public String toString() {
```

Comment:
```suggestion
    @Override
    public String toString() {
```
Would it be better to put an Override?

Reply:
Thanks for the suggestion.